src/components/UserInfo.js

```diff
@@ -1,22 +1,53 @@
 export default class UserInfo {
-  constructor(nameSelector, jobSelector) {
+  constructor(nameSelector, aboutSelector, avatarSelector) {
     this._nameElement = document.querySelector(nameSelector);
-    this._jobElement = document.querySelector(jobSelector);
+    this._jobElement = document.querySelector(aboutSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
 
-  setUserInfo(name, job) {
+  /* -------------------------------------------------------------------------- */
+  /*                              Handle User Info                              */
+  /* -------------------------------------------------------------------------- */
+
+  setUserInfo({ name, about, avatar }) {
+    console.log(name, about, avatar);
     this._nameElement.textContent = name;
-    this._jobElement.textContent = job;
-debugger
+    this._jobElement.textContent = about;
+    this._avatarElement.src = avatar;
   }
 
   // return the current values of the profile text elements
   getUserInfo() {
+    console.log(123);
+    console.log(this._jobElement);
+    console.log(this._nameElement);
+
     const obj = {
       name: this._nameElement.textContent,
-      job: this._jobElement.textContent,
+      about: this._jobElement.textContent,
+      avatar: this._avatarElement.src,
     };
 
     return obj;
   }
+
+  /* -------------------------------------------------------------------------- */
+  /*                             Handle User Picture                            */
+  /* -------------------------------------------------------------------------- */
+
+  setUserAvatar(link) {
```

Comment:
`setUserAvatar` duplicates the code that is inside `setUserInfo`. Please, reuse it there or delete it

Reply:
* You haven't corrected my remark https://github.com/MrWatson0531/se_project_aroundtheus/pull/1#discussion_r1577847447